### DISCUSSION TITLE
Mouse distance

### DISCUSSION THREAD

**Discussion by swinkler01** [rxhanson/Charmstone-Community#11] [Ideas]
I'm finding that when my cursor is near the edge of the screen, I'm unable to activate the far set of apps (same goes for long throws in rectangle pro). Is it possible to use the distance that has been input by the mouse instead of the distance the cursor has traveled on the screen? It might lead to a more consistent experience.

**Comment by rxhanson** [comment]
Thanks for the feedback. As far as I know, it is not possible to get raw mouse movement rather than cursor movement, as in, there's no events or API for that. It is possible to get trackpad input, although there might be some sandbox limitations there. I have been considering adding in trackpad gesture capability for quite some time. Are you a trackpad user; would that be of interest to you? If I go down this path, it might result in a version of the app that is not on the App Store.

**Reply by swinkler01** [reply]
Ah alright, makes sense. I could see gestures making a nice solution (i.e. drag two fingers to access far apps, drag three fingers to access a theoretical third set; or something similar with modifier keys).
